packages/dcos-metrics-collector/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+export GOPATH=$GOPATH:/pkg
+mkdir -p /pkg/src/github.com/dcos/dcos-metrics
+# Create the GOPATH for the go tool to work properly
+cp -R /pkg/src/* /pkg/src/github.com/dcos/dcos-metrics/
```

Comment:
https://github.com/dcos/dcos/pull/778/commits/810f863d2cd7c2513c965fbf531db4e1d87659b6

That didn't seem to work either.